types/react-transition-group/SwitchTransition.d.ts

```diff
@@ -0,0 +1,51 @@
+import { Component } from "react";
+
+export enum modes {
+  out = 'out-in',
+  in = 'in-out'
+}
+
+declare namespace SwitchTransition {
+  interface SwitchTransitionProps {
+    /**
+     * Transition modes.
+     * `out-in`: Current element transitions out first, then when complete, the new element transitions in.
+     * `in-out: New element transitions in first, then when complete, the current element transitions out.`
```

Comment:
I notice the position of the end backquote is incorrect. But the original is same too. 😅 

```suggestion
     * `in-out`: New element transitions in first, then when complete, the current element transitions out.
```


Could you please modify it? I open a PR to the original repository in order to fix the original code.
https://github.com/reactjs/react-transition-group/pull/524